### ISSUE TITLE
Rolling for Weather Event (automatically or not) always outputs public message in chat

### ISSUE THREAD

**Issue by dellatorreadrian** [BernhardPosselt/pf2e-kingmaker-tools#54]
Even when marked to Roll as "Self Roll", "Private Roll", the rolls themselves are kept secret, but the outcome is not.

Print: 
  **First roll:** from Macro
  **Second roll:** automatically when a new day starts

![image](https://github.com/BernhardPosselt/pf2e-kingmaker-tools/assets/72579484/b65c1f02-95d0-48ff-b736-7390d2f10298)

![image](https://github.com/BernhardPosselt/pf2e-kingmaker-tools/assets/72579484/cc783af6-93ab-4a8b-bec2-33dd1a665ee7)

**Comment by BernhardPosselt** [comment]
Note that the chat card mentions Secret. Just tested, works as expected, but it did not work properly when it was set to public roll. That is fixed now.

**Comment by BernhardPosselt** [comment]
No, log into your game with another player. They won't see the chat message.

**Comment by BernhardPosselt** [comment]
Ah dammit, one sec

**Comment by dellatorreadrian** [comment]
Awesome, it's perfect now, thank you very much!
